overall-equipment-effectiveness/workflows/Paper_Machine_Data_Simulator.yaml

```diff
@@ -0,0 +1,1969 @@
+resourceType: Flow
```

Comment:
Why is only one of these given a "Treat value as JSON" check? Same question for the Hard Brand node next to it.

<img width="1141" alt="Screen Shot 2020-09-25 at 6 29 35 PM" src="https://user-images.githubusercontent.com/881769/94321273-20780780-ff5d-11ea-8a20-dbcba16d8b73.png">

Reply:
I did that while problem solving - but it is not necessary - I removed the check.